runtime/lua/vim/filetype/detect.lua

```diff
@@ -1550,7 +1550,7 @@ local patterns_text = {
   ['^SNNS pattern definition file'] = 'snnspat',
   ['^SNNS result file'] = 'snnsres',
   ['^%%.-[Vv]irata'] = { 'virata', { start_lnum = 1, end_lnum = 5 } },
-  ['[0-9:%.]* *execve%('] = 'strace',
+  ['execve%('] = 'strace',
```

Comment:
That is _definitely_ a change that needs to be made upstream first. (Probably a good change, don't get me wrong!)

Reply:
ok, just saw the comment after posting. Sure, I'll try to push it to vim too